armi/physics/neutronics/parameters.py

```diff
@@ -729,7 +729,7 @@ def linPowByPinGamma(self, value):
             units="n/cm^2/s",
             description="Neutron flux above 100keV at hexagon block corners",
             location=ParamLocation.CORNERS,
-            saveToDB=False,
+            saveToDB=True,
```

Comment:
Do these changes have to do with the `Point` change?

Reply:
It does not. I forgot to add this to the description, sorry.

We had talked last week about this PR having this change included as well, since it was a small update. I am happy to separate it out into another PR though, if you'd prefer that.

Reply:
Alternatively, these four parameters could also be moved into the internal plugin, since they are also "associated" with the `Point` class.

Reply:
I'm just asking.  You don't have to change anything. Thanks for the explanation.